src/Adafruit_SPIFlashBase.cpp

```diff
@@ -87,8 +97,9 @@ bool Adafruit_SPIFlashBase::begin(void) {
   // Wait 30us for the reset
   delayMicroseconds(30);
 
-  // Speed up to max device frequency
-  _trans->setClockSpeed(_flash_dev->max_clock_speed_mhz * 1000000UL);
+  // Speed up to max device frequency, or as high as possible
+  _trans->setClockSpeed(
```

Comment:
nice speed limit.

Reply:
Yes, this (combined with #47) should help when folks try to use other devices.